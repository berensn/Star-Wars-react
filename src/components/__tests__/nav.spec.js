import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from '../nav.js';

describe('<Nav />', () => {
  it('should include the correct links', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Nav />
        </MemoryRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
