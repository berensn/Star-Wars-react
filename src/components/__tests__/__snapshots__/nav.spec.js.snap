// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav /> should include the correct links 1`] = `
<div>
  <nav
    className="navElements"
  >
    <span
      className="navItem"
    >
      <a
        aria-current={null}
        className={undefined}
        href="/home"
        onClick={[Function]}
        style={undefined}
      >
        Home
      </a>
    </span>
    <span
      className="navItem"
    >
      <a
        aria-current={null}
        className={undefined}
        href="/people"
        onClick={[Function]}
        style={undefined}
      >
        People
      </a>
    </span>
    <span
      className="navItem"
    >
      <a
        aria-current={null}
        className={undefined}
        href="/planets"
        onClick={[Function]}
        style={undefined}
      >
        Planets
      </a>
    </span>
    <span
      className="navItem"
    >
      <a
        aria-current={null}
        className={undefined}
        href="/species"
        onClick={[Function]}
        style={undefined}
      >
        Species
      </a>
    </span>
    <span
      className="navItem"
    >
      <a
        aria-current={null}
        className={undefined}
        href="/vehicles"
        onClick={[Function]}
        style={undefined}
      >
        Vehicles
      </a>
    </span>
    <span
      className="navItem"
    >
      <a
        aria-current={null}
        className={undefined}
        href="/starships"
        onClick={[Function]}
        style={undefined}
      >
        Starships
      </a>
    </span>
    <span
      className="navItem"
    >
      <a
        aria-current={null}
        className={undefined}
        href="/films"
        onClick={[Function]}
        style={undefined}
      >
        Films
      </a>
    </span>
  </nav>
</div>
`;
